VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -19,6 +19,8 @@
 OSDefineMetaClassAndStructors(VoodooI2CDeviceNub, IOService);
 
 bool VoodooI2CDeviceNub::attach(IOService* provider, IOService* child) {
+    char const *interruptMode = "Polling";
```

Comment:
It might be more explicit/readable to assign a value within the conditions, something like this:

```C++
if (has_gpio_interrupts) {
    ...

     interruptMode = "GPIO";
} else if (has_apic_interrupts) {
    interruptMode = "APIC";
} else {
    interruptMode = "Polling";
}
```

Reply:
Just pushed, this closer to what you were looking for?

Reply:
I think the linter wouldn't like it not initialized as `nullptr`, but yeah, thanks :)

Reply:
The linter didn't seem to care, but I solved a couple other lint things. Will just add nullptr anyways.